awesome/src/widgets/systray.lua

```diff
@@ -8,16 +8,19 @@ local color = require("src.theme.colors")
 local dpi = require("beautiful").xresources.apply_dpi
 local gears = require("gears")
 local wibox = require("wibox")
-require("src.core.signals")
 
+local capi = {
```

Comment:
Is there a particular reason for this? If not please just use awesome.

Reply:
There's none, just copy pasted from AwesomeWM's source and forgot to tidy up